src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -934,6 +934,15 @@ public String getTeamNameForEmail(String email) {
             return Sanitizer.sanitizeForHtml(teamName);
         }
     }
+    
+    public String getUnsanitizedTeamNameForEmail(String email) {
+        String teamName = emailTeamNameTable.get(email);
+        if (teamName == null || email.equals(Const.GENERAL_QUESTION)) {
+            return Const.USER_NOBODY_TEXT;
+        } else {
+            return teamName;
+        }
+    }
```

Comment:
Why is sanitization being mention in a result bundles, question details, etc? These two things are totally unrelated. Need to clean up the sanitization in due course.